### PR TITLE
script: vote to drain the aragon agent wallet of crvrenbtc

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
see #209 for details

deadline to exec this is in roughly ~71 hours

### CODE REVIEW

File: scripts/issue/209/vote_to_drain_agent.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    safe = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
```
Comment (sajanrajdev) [inline scripts/issue/209/vote_to_drain_agent.py:6]: Not sure if missing something but the voting contract logic is hardcoded to require the vote() call from devMulti: https://etherscan.io/address/0xb7acd34643181c879437c2967538d5c0ea42b5d9#code#L686

Comment (gosuto-inzasheru) [reply scripts/issue/209/vote_to_drain_agent.py:6]: i think this is the relevant solidity function: https://etherscan.io/address/0xb935c3d80229d5d92f3761b17cd81dc2610e3a45#code#L1978

**Comment by sajanrajdev** [comment]
Vote data for ID=39 from voting contract: https://etherscan.io/address/0xDc344bFB12522bF3fa58EF0d6b9a41256fc79A1b#readProxyContract
image

File: scripts/issue/209/vote_to_drain_agent.py (hunk 1 above)
Comment (sajanrajdev) [reply scripts/issue/209/vote_to_drain_agent.py:6]: That's true, I somehow ended in the TiemlockWithVoting contract. If interacting directly with the voting contract it should be fine.

**Review by sajanrajdev — APPROVED** [review]
Voting for Vote 39 to transfer crvRenWBTC accumulated to Trops:
image

LGTM

**Pull request merged**